njord/tests/sqlite/update_test.rs

```diff
@@ -0,0 +1,125 @@
+use njord::column::Column;
+use njord::condition::Condition;
+use njord::keys::{AutoIncrementPrimaryKey, PrimaryKey};
+use njord::sqlite::select::SelectQueryBuilder;
+use njord::sqlite::{self};
+use njord::table::Table;
+use njord_derive::Table;
+use std::collections::HashMap;
+use std::path::Path;
+
+#[derive(Table, Clone)]
```

Comment:
Just make sure we dont have any structs here that is not in use. Maybe you could take a look at having these structs in mod.rs instead? So we dont need to define them multiple times. Only have it in the file if it's unique.

Reply:
Actioned ✅